ecosystem/sep-0006.md

```diff
@@ -375,6 +515,87 @@ Example:
+
+### Request
+
+```
+GET TRANSFER_SERVER/withdraw_firm_quote
+```
+
+Request parameters:
+
+Name | Type | Description
+-----|------|------------
+`quote_id` | string | The `id` returned from a `SEP-38 POST /quote` response. The Stellar transaction used to send the asset to the Anchor _must_ have a `created_at` timestamp that is earlier than the quote's `expires_at` attribute.
+`type` | string | Type of withdrawal. Can be: `crypto`, `bank_account`, `cash`, `mobile`, `bill_payment` or other custom values. This field may be necessary for the anchor to determine what KYC information is necessary to collect.
+`dest` | string | The account that the user wants to withdraw their funds to. This can be a crypto account, a bank account number, IBAN, mobile number, or email address.
+`dest_extra` | string | (optional) Extra information to specify withdrawal location. For crypto it may be a memo in addition to the `dest` address. It can also be a routing number for a bank, a BIC, or the name of a partner handling the withdrawal.
+`account` | `G...` string | (optional) The stellar account ID of the user that wants to do the withdrawal. This is only needed if the anchor requires KYC information for withdrawal. The anchor can use `account` to look up the user's KYC information.
+`memo` | string | (optional) A wallet will send this to uniquely identify a user if the wallet has multiple users sharing one Stellar account. The anchor can use this along with `account` to look up the user's KYC info.
+`memo_type` | string | (optional) Type of `memo`. One of `text`, `id` or `hash`.
+`wallet_name` | string | (optional) In communications / pages about the withdrawal, anchor should display the wallet name to the user to explain where funds are coming from.
+`wallet_url` | string | (optional) Anchor can show this to the user when referencing the wallet involved in the withdrawal (ex. in the anchor's transaction history).
+`lang` | string | (optional) Defaults to `en`. Language code specified using [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1). `error` fields in the response should be in this language.
+`on_change_callback` | string | (optional) A URL that the anchor should `POST` a JSON message to when the `status` property of the transaction created as a result of this request changes. The JSON message should be identical to the response format for the [/transaction](#single-historical-transaction) endpoint.
+`amount` | string | (optional) The amount of the asset the user would like to deposit with the anchor. This field may be necessary for the anchor to determine what KYC information is necessary to collect.
```

Comment:
`amount` should be removed here

Reply:
Removed in https://github.com/stellar/stellar-protocol/pull/973/commits/46ff1babb42954d81b5e6ec2682aadc423a1fa82.